spec/mailcheckSpec.js

```diff
@@ -138,6 +138,16 @@ describe("mailcheck", function() {
         var suggestion = mailcheck.suggest('test@yahooo.cmo', domains, secondLevelDomains, topLevelDomains);
         expect(suggestion.domain).toEqual('yahoo.com');
       });
+
+      it("will not offer suggestions for valid 2ld-tld combinations", function() {
+        expect(
+            mailcheck.suggest('test@gmail.co.uk', domains, secondLevelDomains, topLevelDomains)
+        ).toBeFalsy();
+
+        expect(
+            mailcheck.suggest('test@gmail.fr', domains, secondLevelDomains, topLevelDomains)
+        ).toBeFalsy();
+      });
     });
```

Comment:
can you use hotmail or another second level domain for this test please - the idea behind `domains` is that they only have one valid tld and shoudn't be compared so having gmail in both lists is a missleading test case imo